client/src/util/featureFlags/featureFlags.ts

```diff
@@ -25,12 +28,29 @@ export function checkFeatureFlags(): void {
 }
 
 export function getFeatureFlag(key: string): boolean {
+  const params = getParams();
+
+  const paramValue = params?.get(key);
+
+  if (paramValue !== undefined) {
+    setFeatureFlag(key, paramValue || "");
+    return paramValue === ParamBoolean.TRUE;
+  }
+
```

Comment:
This resolves the following bug:
1. Clear local storage
2. Load http://localhost:3000/d/super-cool-spatial.cxg?spatial=true&dl=true
3. `getFeatureFlag("spatial")` actually returns `false` instead of `true`, because the getter is called before the setter

The solution is to prioritize param as the source of truth for feature flag state, and also set the local storage proactively